frontend/src/metabase/ui/components/inputs/Select/Select.styled.tsx

```diff
@@ -29,7 +29,8 @@ export const getSelectOverrides = (): MantineThemeOverride["components"] => ({
       ...getSelectItemsOverrides(theme, size),
       // For epic (metabase#38699)
       dropdown: {
-        background: theme.fn.themeColor("bg-white"),
+        background: "var(--mb-color-background)",
```

Comment:
Various styling for dropdown and select input.

#### Before
![Screenshot 2024-08-01 at 12 40 04 AM](https://github.com/user-attachments/assets/edbebd38-fac5-4fdc-9cef-8689de69a659)

#### After
![image](https://github.com/user-attachments/assets/d841df74-e6d7-478d-93f8-9500c7f6e87f)